ios/keyman/Keyman/Keyman/Classes/KMWebBrowser/WebBrowserViewController.swift

```diff
@@ -51,8 +51,7 @@ class WebBrowserViewController: UIViewController, UIWebViewDelegate, UIAlertView
       observer: self,
       function: WebBrowserViewController.keyboardPickerDismissed)
 
-    webView.delegate = self
-    webView.scalesPageToFit = true
```

Comment:
There is no corresponding property on `WKWebView`.  That said, what was its original role?  Well...

https://github.com/react-native-webview/react-native-webview/issues/447#issuecomment-477201365

This comment (and others) indicate that it served to automatically model the classic `<meta name="viewport" content="width=device-width, initial-scale=1 />` HTML header tag functionality.  This may have been needed years ago, but sites worth their salt _these_ days should have that in their code already.

So... we should be completely safe to drop this, unless we want to try tag injection:  https://stackoverflow.com/a/26583062

Reply:
... I don't know how to answer that question.

Reply:
I am not sure we want to do the tag injection automatically anyway so let's see how this goes.